### DISCUSSION TITLE
mrec cannot play big size file

### DISCUSSION THREAD

**Discussion by cquintela** [Ylianst/MeshCentral#5751] [Issues] [answered]
Hello, I need some help. I recorded a session of about two hours, the file was 1.5 gigabytes. Now I can't play it, I load it to play and nothing happens. Is this a limitation of my server or a system size limitation? In my config.json the limit is 3 gigabytes. The server log and the browser console don't show any errors.

**Comment by si458** [comment]
one thing to try is to download the mcrec file, then visit `https://MYSERVER.COM/player.htm` and open the file locally,
its possible its trying to stream the 1.5gb to your browser which is stupid as thats huge!

**Reply by cquintela** [reply]
I've installed Meshcentral locally to run both the player and the file, but no luck. As soon as I load the file into the player and press OK, it returns to the standard player page as if it were loading a new file. I think the file must have been corrupted because of its size, usually files are 200-300 megabytes long, even with a lot of user screen changes. But thanks for your attention and quick reply! I'll give it a few more tries and if I get anything, I'll update the post.

**Reply by si458** [reply, marked as answer]
do you have `index` set as true with `sessionrecording` ?
it might be the server is still BUSY indexing the file as 1.5gb is a HUGE file so it might take some time,
maybe give it an hour then try again?
you can also try visiting the player page, then opening the browser developer tools and check the console tab and see if the is any errors while it trys to load it

**Reply by cquintela** [reply]
Oh, yes, I have it. The console shows no error, but I will wait some time, the index can be a good guess.

**Reply by si458** [reply]
if i remember it should put the string `MeshCentralMCNDX` either at the beginning or end of the file? thats how you know its finished indexing

**Reply by cquintela** [reply]
Bingo! I thought that when the file was available for download it was already "ready", I've been trying to find a solution for an hour now. I downloaded the file from the server again after speaking to you and the file I downloaded worked! Thank you very much.

**Reply by si458** [reply]
no worries, just an idea 💡
can you open a github issue (enhancement) and just say it would be a good idea, we have a status icon so you know IF i file has finished indexing or not...
